Client/src/Pages/Login/index.jsx

```diff
@@ -78,7 +78,10 @@ const Login = () => {
         // validation errors
         createToast({
           variant: "info",
-          body: error.details[0].message,
+          body:
+            error && error.details && error.details.length > 0
+              ? error.details[0].message
+              : "Error validating data.",
```

Comment:
oh wait I don't think error.details.length > 0 is relevant here

Reply:
woops, speed merged it, go ahead and open a fix PR

Reply:
Disregard this, I just had a brain fart 🤣 Length will still be > than 0 when we're accessing the 0 index.

Reply:
Good ol off by 1 :stuck_out_tongue: